Orso.Arpa.Api/Startup.cs

```diff
@@ -149,11 +154,25 @@ public void ConfigureServices(IServiceCollection services)
 
             ConfigureCors(services);
 
+            useCookies = Configuration.GetSection("IdentityConfiguration").GetValue<bool>("UseCookies");
+
             if (_hostingEnvironment.IsProduction())
             {
                 _ = services.AddApplicationInsightsTelemetry();
             }
             _ = services.AddMediatR(typeof(LoginUser.Handler).Assembly);
+
+            if (useCookies)
+            {
+                services.AddAuthentication(options =>
+                {
+                    options.DefaultAuthenticateScheme = IdentityConstants.ApplicationScheme;
+                    options.DefaultChallengeScheme = IdentityConstants.ApplicationScheme;
+                    options.DefaultSignInScheme = IdentityConstants.ExternalScheme;
+                })
+                .AddIdentityCookies();
```

Comment:
configuring authentication scheme and cookies is necessary since using AddIdentityCore requires more direct control to the setup

If we were using AddIdentity then authentication schemes and cookies would be configured automatically without a need for these initialisations